apps/hip/templates/includes/footer.html

```diff
@@ -1,6 +1,6 @@
 <footer class="footer-section-hip has-text-centered pt-5">
   <div class="footer-section-blue-hip">
-    <p class="pt-5">Copyright 2025 by The City of Philadelphia. All Rights Reserved.</p>
+    <p class="pt-5">Copyright 2015 - 2025 by The City of Philadelphia. All Rights Reserved.</p>
```

Comment:
Looking at the wayback machine, it looks like 2015 was when the site was officially spun up and so there could be content that would need to be protected.